.github/workflows/ci-aarchxx-cross.yml

```diff
@@ -163,7 +163,7 @@ jobs:
 
```

Comment:
@derekbruening Could you please remove x86-32-ubuntu22 and x86-64-ubuntu22 from the list of required workflows?